packages/web/src/builders/package/package.impl.ts

```diff
@@ -98,12 +101,16 @@ export function run(
             }),
             postcss({
               inject: true,
-              extract: false,
+              extract: options.extractCss,
```

Comment:
Allow users to control CSS extraction through `--no-extract-css` or `--extract-css` option (or specify option in workspace.json).